.github/workflows/release.yml

```
@@ -12,126 +12,174 @@ permissions:
+        id: get_version
+        run: |
+          CURRENT_VERSION=$(grep "^version = " Cargo.toml | cut -d'"' -f2)
+          echo "version=$CURRENT_VERSION" >> $GITHUB_OUTPUT
+          echo "Current version: $CURRENT_VERSION"
+
+      - name: Check if version changed
+        id: check_version
+        run: |
+          CURRENT_VERSION="${{ steps.get_version.outputs.version }}"
+          PREVIOUS_VERSION=$(git show HEAD~1:Cargo.toml | grep "^version = " | cut -d'"' -f2 2>/dev/null || echo "")
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)